modules/renter/hostdb/scan.go

```diff
@@ -175,6 +176,7 @@ func (hdb *HostDB) threadedScan() {
 			for _, randIndex := range hostOrder {
 				hdb.scanHostEntry(entries[randIndex])
 			}
+			hdb.save()
```

Comment:
I don't think this `save` accomplishes anything, since the scans run concurrently.